### DISCUSSION TITLE
Official High Quality Bedrock Icons for us!

### DISCUSSION THREAD

**Discussion by breezefps** [Mojang/minecraft-editor#122] [Feedback]
Hi guys! Please provide as the new Icons added to the Bedrock version, it's easy to get on the java version, but on the bedrock one it's hard! For as content creators, this changes everthing!

**Comment by minecraftchloe** [comment]
@breezefps Which Bedrock icons are you referring to specifically that you want the high quality versions of?

**Reply by breezefps** [reply]
Those that were announced here Chloe :)

https://www.minecraft.net/en-us/article/our-icons-are-changing

**Reply by minecraftchloe** [reply]
Gotcha!! Thank you! I thought you were referring to some Editor icons - let me do some sleuthing internally quick to see if I can point you to where to access these, if they're publicly available.
